server/views/pages/calculation/checkInformation.njk

```diff
@@ -31,8 +32,21 @@
 {% block content %}
     {% set validationErrors = model.validationErrors %}
     {% include "../partials/customErrorBanner.njk" %}
+    {% if featureToggles.ersed and not model.viewJourney and not model.isActiveRecallOnly() %}
```

Comment:
I think the notification banner should also appear in the view journey. See AC3.